core/consensus/grandpa/environment.hpp

```diff
@@ -62,39 +62,26 @@ namespace kagome::consensus::grandpa {
     /**
      * Make catch-up-response
      */
-    virtual outcome::result<void> onCatchUpResponsed(
+    virtual outcome::result<void> onCatchUpRespond(
```

Comment:
Pls, explain

Reply:
Hm, it seems more natural for me because a catch-up response is a term from the spec and it reads as 'on catch-up response [received]', but with 'respond' it's trickier, because if we read is as 'on(when) catch up responds' it makes no sense, it actually would mean something like 'on(when) [a peer] responds [with] catch-up [message]'. I didn't actually think about it, my eyes just stumbled while reading the code and I thought that response would fit a bit more neatly. It's a very minor thing, you most certainly can leave it any way you see appropriate.